releases/0.6/index.md

```diff
@@ -1,8 +1,181 @@
+cloud environments. Through the configuration file the logging system is
+infinitely user configurable. The system supports logging to any combination of
+`stdout`, `stderr`, file, and rolling file targets. Those logs are independently
+filterable by their level of severity to tailor logging output to only the logs
+needed in each deployment.
+
+### REST API Security
+
+#### Additional Authentication Methods
+
+Splinter now supports two additional authentication methods: OAuth2 and
+Cylinder JWT (JSON Web Token).
+
+OAuth2 support allows the Splinter REST API to integrate with existing
+enterprise solutions, such as Azure, Google and Github, to provide
+authentication services. The identity returned from an OAuth2 provider can then
+be used for access control.
+
+Cylinder JWT support allows other daemons and CLI tools to authenticate with the
+Splinter REST API by using a signing key to create a JWT. Like challenge
+authorization, the token includes a public key and a signature that can be used
+to verify the signer.  The public key can be used then be used for access
+control.
+
+Biome authentication is still available, though still not recommended for
+production deployments.
+
+#### Authorization
+
+Splinter also supports two access control method for authorizing a user's access
+to the various REST API endpoints.
+
+Each Splinter node may include a list of public keys in a configuration file.
+Keys added to the `allow_keys` file will be allowed access to all REST API
+endpoints.  This list is recommended for node administrators only.
+
+Both keys and identities (Biome and OAuth2) may have their access configured
+through Splinter's role-based access control system. Access can be granted via
+roles which, in turn, have a set of permissions associated with them.
+
+All authorization configuration is limited to a single node, and does not affect
+any other members of a given circuit.
+
+For more information, see the [Configuring REST API Authorization]({% link
+docs/0.6/howto/configuring_rest_api_authorization.md %}) guide.
+
+### And More
+
+* **Circuit Display Name** ability to add a human readable name to the circuit
+  definition
+* **Circuit Templates** simplifies constructing common circuits
+* **Metrics** basic metrics collection, including Scabbard information
+* **Biome profiles** storage for user profiles, including basic information
+  retrieved from an OAuth2 provider
+
+#### Documentation
+
+This release includes documentation for Splinter concepts, features, and tasks
+(under development), API reference guides, CLI man pages, and a comprehensive
+walkthrough for the Gameroom example application.
+
+* [Splinter documentation]({% link docs/0.6/index.md %})
+
+* [splinterd REST API Reference](/docs/0.6/api/)
+
+* [Rust crate:splinter API
+  documentation](https://docs.rs/splinter/latest/splinter/)
+
+* [Rust crate:scabbard API
+  documentation](https://docs.rs.splinter/latest/scabbard)
```

Comment:
I think this link should be https://docs.rs/scabbard/latest/scabbard/

Reply:
Fixed